sources/vega/commands/v1/validator_commands.proto

```diff
@@ -75,3 +75,14 @@ message ChainEvent {
     vega.StakingEvent staking_event = 1005;
   }
 }
+
+// A transaction to allow validator to rotate their vega keys
+message KeyRotateSubmission {
+  // Index of the new public key
+  uint64 key_number = 1;
+  // Target block at which the key rotation will take effect on
+  uint64 target_block = 2;
+  int64 time = 3;
+  // New public key to rotate to
```

Comment:
Minor: comment should be "_Hash_ of the public key to rotate too". Also, again minor, but maybe `next_pub_key_hash` would be a better name for things?

Reply:
I think `new_` works just fine tbh. But changed the comment. Thanks